rest_framework/relations.py

```diff
@@ -271,7 +272,7 @@ def __init__(self, view_name=None, **kwargs):
             self.view_name = view_name
         assert self.view_name is not None, 'The `view_name` argument is required.'
         self.lookup_field = kwargs.pop('lookup_field', self.lookup_field)
-        self.lookup_url_kwarg = kwargs.pop('lookup_url_kwarg', self.lookup_field)
+        self.lookup_url_kwarg = kwargs.pop('lookup_url_kwarg', self.lookup_url_kwarg)
```

Comment:
This changes the behavior in an unintentional way. Currently we default to the same value as `lookup_field` (either as set on the class, or as provided as a keyword argument). This change removes that.